pkg/rfc8693/claims.go

```diff
@@ -44,37 +46,56 @@ func parseCEL(input string) (cel.Program, error) {
 	return prog, nil
 }
 
+type subToCEL map[string]cel.Program
+
 // ClaimMappingStrategy represents a mapping from external identity claims to DMV claims.
 type ClaimMappingStrategy struct {
-	mappings map[string]cel.Program
+	mappings map[string]subToCEL
 }
 
 // NewClaimMappingStrategy creates a ClaimMappingStrategy given a mapping of desired DMV claims to CEL expressions.
-func NewClaimMappingStrategy(mappingExprs map[string]string) (ClaimMappingStrategy, error) {
-	mappings := make(map[string]cel.Program, len(mappingExprs))
+func NewClaimMappingStrategy(issuers []fositex.Issuer) (ClaimMappingStrategy, error) {
+	itom := make(map[string]subToCEL, len(issuers))
 
-	for k, e := range mappingExprs {
-		prog, err := parseCEL(e)
-		if err != nil {
-			return ClaimMappingStrategy{}, err
+	for _, issuer := range issuers {
+		mappingExprs := issuer.ClaimMappings
+
+		// If there are no mappings, skip this issuer.
+		// Should we have a default mapping?
+		if len(mappingExprs) == 0 {
```

Comment:
I'm actually fine with the idea of no mappings just meaning we issue a token with `sub` and `iss` and `aud` and basically nothing else. What do you think?